dotCMS/src/main/java/com/dotmarketing/image/filter/ImageFilter.java

```diff
@@ -55,19 +59,32 @@ private String getUniqueFileName(File file, Map<String, String[]> parameters, St
 			}
 
 			while (it.hasNext()) {
-				Map.Entry pairs = (Map.Entry) it.next();
-				String key = (String) pairs.getKey();
-				String val = ((String[]) pairs.getValue())[0];
+				Map.Entry<String, String[]> pairs = it.next();
+				String key = pairs.getKey();
+				String val = pairs.getValue()[0];
 
-				for (String x : acceptFilter) {
-					if (key.startsWith(x)) {
+				for (String filterName : acceptFilter) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Local variable 'filterName' could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4761083)